### ISSUE TITLE
Speed up re-render

### ISSUE THREAD

**Issue by CJ-Wright** [conda-forge/conda-smithy#723]
It would be great if re-render could be faster (comparing against 2.4.5). This would help speed up the bot, which currently re-renders every feedstock it PRs.

**Comment by CJ-Wright** [comment]
xref: https://github.com/regro/cf-scripts/pull/115#issuecomment-377971152

**Comment by jakirkham** [comment]
Agreed. Along these lines we should try to do some benchmarking of steps so we can identify the slow parts. Recommendations on how we do this?

**Comment by CJ-Wright** [comment]
Maybe try [asv](https://asv.readthedocs.io/en/latest/)?
I guess a test setup could be to take a repo then git checkout an old re-render and run a re-render.

**Comment by jakirkham** [comment]
FYI `conda-build` has an `asv` setup now. ( https://github.com/conda/conda-build/pull/2859 )
